### ISSUE TITLE
v2.0.0-pre33: Tor Settings seem not to stick and seems to impact price feature

### ISSUE THREAD

**Issue by moritzwietersheim** [cryptoadvance/specter-desktop#2279]
- I went through the Tor functionality on a Electrum setup. 
- The Tor settings seem to jump back to "disabled" and not stay on "Built in"
- Only stays on "Built in" if "Force all external calls over Tor" is selected.

- Also the Price feature does strange things on Tor. 
- Doesnt work on Bitstamp, but on Coindesk, but jumps back to Bitstamp.

- Logs attached below.
- If higher resolution video is needed, pls let me know.

https://user-images.githubusercontent.com/47259243/222628383-edcca59b-5266-44cb-b77d-622b766cd5a1.mov

[230302 1101 specterApp copy.log](https://github.com/cryptoadvance/specter-desktop/files/10878057/230302.1101.specterApp.copy.log)

**Comment by k9ert** [comment]
yes, bitstamp is known to not work with core. We need to put a warning in there and then also switch it off as price-provider.
https://github.com/cryptoadvance/specter-desktop/issues/1554
